contracts/strategies/allocation/alt-registry/README.md

```diff
@@ -0,0 +1,47 @@
+Spec: Application from alternate registry
+---------------------------------
+
+## Overview 
+Some builders have flagged that they want the ability to use alternate "registries" to determine whether or not a potential recipient is eligible to receive funds from their pool. This strategy is designed to test that specific assumption, with basic mechanisms for the non-recipient components. 
+
+## Component Quickview
+As laid out in the [components overview](https://docs.google.com/document/d/1qoOP07oMKzUCyfb4HbnyeD6ZYEQa004i5Zwqoy7-Ox8/edit), each allocation strategy consists of some key components. This is a quick overview of the relevant components for this strategy.
+- **Recipient eligibility**: alternate registry
+- **Recipient approval**: programmatic
+- **Voter eligibility**: pool admin
+- **Voting method**: simple yes/no
+- **Calculation method**: simple yes/no
+- **Allocation shape**: discrete amount
+
+## Spec
+### Custom Variables
+This strategy will need the following custom variables:
+- `poolOpen` - boolean value that represents whether the pool is accepting new applications
+- `alternateRegistry` - location of alternate registry of eligible recipients
```

Comment:
I think we should dial in on one specific example like Hypercerts, Passport(when onchain is ready), Givith or Optimism.

EAS is another option which I am very fond of experimenting with.

Reply:
Based on how the protocol has developed, will it still need these custom variables? I don't think so

Reply:
@0xZakk `poolOpen` might not be relevant, but we need somewhere to account for how the strategy will check for eligibility, right?